lib/widgets/selection.dart

```diff
@@ -2342,7 +2342,7 @@ class SelectAllSelectionAction<T extends Content> extends StatelessWidget {
       final first = all.first;
       assert(all.every((el) => el.runtimeType == first.runtimeType));
```

Comment:
```suggestion
      assert(all.every((el) => el.type == first.type));
```
Let's not use the `runtimeType` here while we are a